### DISCUSSION TITLE
Burst skill

### DISCUSSION THREAD

**Discussion by thegreatestbrian** [Melledy/Grasscutter#974] [Q&A] [answered]
how can i use q skill without countdown? 
![Screenshot 2022-05-18 185250](https://user-images.githubusercontent.com/105790655/169032592-8fc2577a-8b9e-4227-8e1e-de978bd641ad.png)

**Comment by Birdulon** [comment, marked as answer]
`/stats cdr 100%`
Note that being hit by an Abyss Herald attack will reset it.

**Reply by thegreatestbrian** [reply]
thanks :D
